docs/src/assets/scss/ads.scss

```diff
@@ -4,6 +4,15 @@
     }
 }
 
+.docs-ad {
```

Comment:
we had a wrapper in the main page to avoid shift of content when added popsup. Seems like docs didn't have it so added this.